finetuner/client/client.py

```diff
@@ -186,7 +186,15 @@ def stream_run_logs(self, experiment_name: str, run_name: str) -> Iterator[str]:
         response = self._handle_request(url=url, method=GET, stream=True)
         for entry in response.iter_lines():
             if entry:
-                yield entry.decode('utf-8', errors='ignore')
+                decoded_message: str = entry.decode('utf-8', errors='ignore')
+                sep_pos = decoded_message.find(': ')
+                if sep_pos != -1:
```

Comment:
can you elaberate what is this doing?

Reply:
The EventSourceResponse class generates different kinds of log message (e.g., "event" and "data" messages). It adds this type at the beginning of the message, e.g., "data: Finetuning ...". This code snippet divides the type and the message by splitting it at the first occurrence of ": " and only returns the message itself as log message.

Reply:
what if `sep_pos` is None?

Reply:
The `find` function returns -1 if it is not present. Therefore, I check for -1